### ISSUE TITLE
Publish Python 3-compatible release of pypi-release

### ISSUE THREAD

**Issue by ghost** [pinterest/pinrepo#5]
Would it be possible to publish a new version of pypi-release to public PyPi?

Right now 1.0.1 is the latest release of pypi-release available in public PyPi.  It is not Python 3-compatible, failing with
```text
Traceback:
  …
  File "…/site-packages/pypi_release/pypi_release.py", line 73
    print "wait until %s is available in S3..." % key
          ^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print("wait until %s is available in S3..." % key)?
```

However, the current HEAD of `master` (0a80230beb758acc7ca81e798bdd2a5d616bcd72 as of this writing) works fine in Python 3.  This was tested with CPython 3.9 in `ubuntu-latest` in GitHub Actions.

**Comment by jparise** [comment]
The previous PyPI maintainer is no longer a Pinterest employee so I've started the process to bring it back under our control so we'll be able to make new releases.

https://github.com/pypa/pypi-support/issues/1108

**Comment by ghost** [comment]
@jparise Any luck getting ownership of the PyPi project?

**Comment by jparise** [comment]
Nothing yet.

**Comment by jparise** [comment]
The ownership transfer just went through, and I just published [version 1.0.2](https://pypi.org/project/pypi-release/1.0.2/) to PyPi.
